packages/loopring_v3/operator/state.py

```diff
@@ -844,6 +875,58 @@ def cancelOrder(self,
                                     None)
         return cancellation
 
+    def internalTransfer(self,
+                    exchangeID, operatorAccountID, accountFromID, accountToID,
+                    transTokenID, amountRequested, feeTokenID, fee, label):
+
+        feeValue = roundToFloatValue(fee, Float16Encoding)
+
+        # Update account From
+        rootBefore = self._accountsTree._root
+        accountBefore = copyAccountInfo(self.getAccount(accountFromID))
+        nonce = accountBefore.nonce
+        proof = self._accountsTree.createProof(accountFromID)
+
+        balanceUpdateF_From = self.getAccount(accountFromID).updateBalance(feeTokenID, -feeValue)
+        balanceT = int(self.getAccount(accountFromID).getBalance(transTokenID))
+        uAmountTrans = int(amountRequested) if (int(amountRequested) < balanceT) else balanceT
```

Comment:
I think this `min` logic can be removed because the circuit doesn't do this any more. The circuit will fail either way if this happens so in the end it doesn't really matter.